scripts/setup_webserver.sh

```diff
@@ -330,10 +333,11 @@ EOF
   fi # if [ "$webServerType" = "apache" ];
 
    # php config 
+   PhpVer=$(get_php_version)
```

Comment:
Maybe nitpicking, but why would you call this again and repeat the same assignment, when it's already done in line 91?

Reply:
I have removed the duplicate line.